### ISSUE TITLE
Implement Orchid menu functionality and/or assign active class and ARIA to current page

### ISSUE THREAD

**Issue by erinchambers** [CDRH/austen#86]
This is a "should"—not a "must"—re: WCAG 2.1

From Deque's summary of navigation accessibility:
• A navigation list SHOULD include a visible method of informing users which page within the navigation list is the currently active/visible page.  • A navigation list SHOULD include a method of informing blind users which page within the navigation list is the currently active/visible page. 

[We implemented this in Orchid](https://github.com/CDRH/orchid/blob/dev/app/views/layouts/body/_navbar.html.erb) by adding two things to the current page's navbar link:
1. Add `class="active"` to the list item (`<li>`)
2. Add `aria-current="page"` to the link 

...however, the code we used for Orchid doesn't work for Austen Said. Looks like there's already some code in `application.html.erb` that's trying to add `aria-current`, but it's not working.

For reference, see the [Chesnutt dev site](https://cdrhrailsdev.unl.edu/chesnutt) and the corresponding [navbar partial](https://github.com/CDRH/chesnutt/blob/dev/app/views/layouts/body/_navbar.html.erb).
